config/application.yml

```diff
@@ -1,6 +1,8 @@
 level_limit: 3
 api_key: <%= ENV['API_KEY'] || 'PUT_YOUR_API_KEY_HERE' %>
 elasticsearch_server: <%= ENV['SEARCH_SERVER_ES7'] || 'http://localhost:9200' %>
```

Comment:
The fallback value here already includes a port. Is it not possible to specify 443 the same way? From reading the documentation it seems like you should be able to set the elasticsearch_server to include the port, and it should work as expected.

Reply:
So in my initial testing it didn't appear to have an impact, but explicitly adding the port as `https://foo.example.com:443` does seem to have the desired effect. Being able to bypass the TLS validation would still be useful though. I'll update this PR.

Reply:
Yup, that change seems perfectly OK, but I feel having two ways to declare ports would be a bit confusing, although Mongoid itsels seems to support that, so perhaps I shouldn't be concerned.